app/uk/gov/hmrc/cataloguefrontend/platforminitiatives/PlatformInitiativesListPage.scala.html

```diff
@@ -0,0 +1,143 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.mvc.Call
+@import uk.gov.hmrc.cataloguefrontend.DateHelper._
+@import uk.gov.hmrc.cataloguefrontend.platforminitiatives._
+@import uk.gov.hmrc.cataloguefrontend.util.MarkdownLoader
+@import uk.gov.hmrc.cataloguefrontend.ViewMessages
+@import helper._
+@import scala.concurrent.Future
+@import java.net.URL
+@import scala.util.Try
+
+@this(viewMessages: ViewMessages)
+
+@(initiatives          : Seq[PlatformInitiative],
+displayChart           : Boolean,
+displayProgress        : Boolean,
```

Comment:
do we support all 4 possibilities? looking at the links, it's just two - could be represented with `display = Chart | Progress` data type